Mds/Cache.php

```diff
@@ -60,7 +60,7 @@ public function get( $name )
 
 	public function put( $name, $value, $time = 1440 )
 	{
-		$cache = array( 'value' => $value, 'valid' => time() + ( $time*60 ) );
```

Comment:
We need to ensure that the minimum supported PHP version is updated in the docs, 
PHP 5.4+ is more than reasonable, but it may be worth checking